compiler/one-cmds/one-infer

```diff
@@ -208,6 +211,14 @@ def main():
     # run backend driver
     _utils._run(infer_cmd, err_prefix=ntpath.basename(driver_path))
 
+    # run post process script if it's given
+    if _utils._is_valid_attr(args, 'post_process'):
+        # NOTE: the given python script will be executed by venv of ONE
+        python_path = sys.executable
+        post_process_command = [python_path] + getattr(args,
+                                                       'post_process').strip().split(' ')
```

Comment:
```suggestion
        post_process_command = [python_path] + getattr(args, 'post_process').strip().split()
```
(trivial) `split` has default parameter - space

Reply:
I'd like to specify the token. :) thank you for pointing it. :smile: